src/components/Scrollable/Scrollable.tsx

```diff
@@ -31,6 +31,8 @@ export interface Props extends React.HTMLProps<HTMLDivElement> {
   shadow?: boolean;
   /** Slightly hints content upon mounting when scrollable */
   hint?: boolean;
+  // Fires when user reached the bottom of the scroll area
```

Comment:
We need the specific comment style for the type to be correctly parsed
```suggestion
  /** Fires when user reached the bottom of the scroll area */
```

Reply:
```suggestion
  /** Called when scrolling to the bottom of the scroll area */
```

Reply:
Great suggestion, thanks!

Reply:
`scrolling to` feels like an ongoing action to me, whereas I think the intention is to indicate when bottom has been reached? What about:

```suggestion
  /** Called when scrolled to the bottom of the scroll area */
```